version/version.go

```diff
@@ -0,0 +1,3 @@
+package version
+
+const TerraformProviderJujuVersion = "0.8.0"
```

Comment:
We don't need a new package for this - it's test only. Can go in provider_test.go

Reply:
I thought in the future we can do something with this, given that we have another constant sitting in the Makefile, maybe we can eliminate that one as well.